### PR TITLE
Automate the infrastruture pieces

### LINKED ISSUE #116: Automate the infrastructure needs
The entry point for Kraken should be as simple as possible. We leverage elasticsearch, kibana and grafana for storing and  visualizing metrics. While having a hosted infra is needed for long term storage, there are cases where there's no access to infra, we need to support a lean way to automatically spin up infra containers using podman-compose to enable users to leverage the features without much effort.

### PR DESCRIPTION
**Pull request by chaitanyaenr**
### Description
This commit:
- Adds support to automate the infrastructure pieces leveraged by Kraken
  including Elasticsearch and Cerberus.
- Adds a grafana template to visualize the metrics captured by kube-burner.
- Adds a config that can be used to discover all the infra pieces automatically
  without having to tweak the configuration.

Fixes #116

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -14,3 +14,4 @@ python-openstackclient
 gitpython
 paramiko
 setuptools
+podman-compose
```
Comment (paigerube14) [inline requirements.txt:17]: Can we also add docker-compose

File: README.md [hunk 2]
```diff
@@ -15,6 +15,17 @@ See the [getting started doc](docs/getting_started.md) on support on how to get
 After installation, refer back to the below sections for supported scenarios and how to tweak the kraken config to load them on your cluster
 
 
+### Setting up infrastructure dependencies
+Kraken indexes the metrics specified in the profile into Elasticsearch and visualizes them in Grafana in addition to leveraging Cerberus for understanding the health of the kubernetes/OpenShift cluster under test. More information on the features is documented below. The infrastruture pieces can be easily installed by running:
+
+```
+$ cd kraken
+$ podman-compose up
+```
```
Comment (paigerube14) [inline README.md:24]: found that there is a `podman-compose down` should we add that to this doc?

File: README.md [hunk 3]
```diff
@@ -15,6 +15,17 @@ See the [getting started doc](docs/getting_started.md) on support on how to get
...
+
+This will spin up Elasticsearch, Grafana and Cerberus containers on the host on which you are running Kraken.
+
```
Comment (paigerube14) [inline README.md:27]: Ravi and I talked about this offline, but I had issues with the size of my linux ec2 machine. Might be helpful to mention the [doc](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/recognize-expanded-volume-linux.html) we used to get around this issue

File: requirements.txt (hunk 1 above)
Comment (chaitanyaenr) [reply requirements.txt:17]: Thanks to pointers from @paigerube14 around not needing to use the host network for the containers. Both podman-compose, docker-compose should work now.

File: README.md (hunk 2 above)
Comment (chaitanyaenr) [reply README.md:25]: Fixed it, thanks.

File: README.md (hunk 3 above)
Comment (chaitanyaenr) [reply README.md:29]: Fixed it, thanks.

File: README.md [hunk 4]
```diff
@@ -15,6 +15,19 @@ See the [getting started doc](docs/getting_started.md) on support on how to get
 After installation, refer back to the below sections for supported scenarios and how to tweak the kraken config to load them on your cluster
 
 
+### Setting up infrastructure dependencies
+Kraken indexes the metrics specified in the profile into Elasticsearch and visualizes them in Grafana in addition to leveraging Cerberus for understanding the health of the kubernetes/OpenShift cluster under test. More information on the features is documented below. The infrastruture pieces can be easily installed, uninstalled by running:
+
+```
+$ cd kraken
+$ podman-compose up or $ docker-compose up    # Spins up the containers specified in the docker-compose.yml file present in the run directory
```
Comment (paigerube14) [inline README.md:23]: Might be worth mentioning here that this command will continuously run/don't end it till after kraken execution

Comment (chaitanyaenr) [reply README.md:23]: Fixed it, thanks.

**Review by paigerube14 — APPROVED** [review]
LGTM

**Pull request merged**